stable/jenkins/values.yaml

```diff
@@ -142,6 +142,9 @@ Agent:
       memory: "256Mi"
   # You may want to change this to true while testing a new image
   AlwaysPullImage: false
+  # Controls how slave pods are retained after the Jenkins build completes
+  # Possible values: Always, Never, OnFailure
```

Comment:
is Default a possible value as well?

Reply:
No, 'Default' value is not applicable in this context. Line 110 in config.yaml defines PodRetention on Kubernetes section level and line 97 overrides this value on pod template level (which doesn't really make sense, therefore usage of 'Default' value there causing to use value from line 110).